src/scripts/main.js

```diff
@@ -1 +1,61 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
```

Comment:
The first promise will always be rejected after 3 seconds if the click event doesn't occur before the timeout. Consider whether this behavior is intended, as it may lead to unexpected rejections if the user doesn't click within the time limit.